packages/shared/src/hooks/feed/useAutorotatingAds.ts

```diff
@@ -0,0 +1,113 @@
+import { useCallback, useContext, useEffect, useMemo } from 'react';
+import {
+  useQuery,
+  useQueryClient,
+  type InfiniteData,
+  focusManager,
+} from '@tanstack/react-query';
+import {
+  useInView,
+  type InViewHookResponse,
+} from 'react-intersection-observer';
+import { useLogContext } from '../../contexts/LogContext';
+import { ActiveFeedContext } from '../../contexts';
+import { useFeature } from '../../components/GrowthBookProvider';
+import { featureAutorotateAds } from '../../lib/featureManagement';
+import type { Ad } from '../../graphql/posts';
+import { apiUrl } from '../../lib/config';
+import { generateAdLogEventKey } from './useLogImpression';
+import { disabledRefetch } from '../../lib/func';
+import { RequestKey } from '../../lib/query';
+
+export type InViewRef = InViewHookResponse['ref'];
+
+export const useAutorotatingAds = (
```

Comment:
I got triggered 🙈 (also the file name and everywhere else)

```suggestion
export const useAutoRotatingAds = (
```

Reply:
😂

Reply:
Fixed

Reply:
File name is still lower case 🙈

Reply:
Now it should be fixed...